### ISSUE TITLE
Reloading page after closing server still shows action

### ISSUE THREAD

**Issue by tlienart** [asprionj/LiveServer.jl#1]
Actions:

1. serve
2. shut it down with CTRL+C
3. hit refresh in the browser this shows

```julia-repl
julia> serve()
Starting live-server on 0.0.0.0:8000...
Adding file './index.html' to watcher...
Adding file './css/master.css' to watcher...
WebSocket request from /...
WebSocket connections for file './index.html':
wsi.io = T0  ⏸    0↑🔒    0↓🔒   0s 0.0.0.0:8000:8000 ≣16

✓ server closed gracefully.

julia> Adding file './index.html' to watcher...
Adding file './css/master.css' to watcher...
```

So this shows that the FileWatcher is not stopped properly.

**Edit**: we should consider using [watch_folder](https://docs.julialang.org/en/v1/stdlib/FileWatching/index.html#FileWatching.watch_folder) which can be stopped with [unwatch_folder](https://docs.julialang.org/en/v1/stdlib/FileWatching/index.html#FileWatching.unwatch_folder)

**Comment by tlienart** [comment]
Ok so we're not properly closing websockets, tasks, and HTTP.listen (either all or some). 

A way to see it:

* start liveserver
* stop it
* go to localhost:8000 (still there, no problem)
* exit julia (ok now it's stopped)

**Comment by tlienart** [comment]
(in the PR #7 this shows a 404 by the way which is better imo)

**Comment by asprionj** [comment]
For the WebSockets, we should use `close(ws)` or `close(ws.io)` on each element of each value in `WS_HTML_FILES`, not quite sure about which one. The former eventually calls the latter but attempts to somehow "cleanly" close the WS. Do we need that? We're stopping the server anyway...

**Comment by tlienart** [comment]
this is fixed in #7 see comment there

**Comment by asprionj** [comment]
I think by now we can say that this is an issue with HTTP.jl, see #10...? If you agree, please close this issue.
